packages/stores/src/account/types.ts

```diff
@@ -61,4 +62,15 @@ export type RegistryWallet = Wallet & {
    * the notifications button.
    */
   features: Array<"notifications">;
+
+  signOptions?: SignOptions;
 };
+
+export type AccountStoreWallet<Injects extends Record<string, any>[] = []> =
```

Comment:
Moving some stuff around here?

Reply:
We can disregard this. These were changes from this PR to `master`